pkg/lib/accountmanagement/service.go

```diff
@@ -184,3 +277,220 @@ func (s *Service) FinishAdding(input *FinishAddingInput) (*FinishAddingOutput, e
 
 	return &FinishAddingOutput{}, nil
 }
+
+type ResendOTPCodeInput struct {
+	UserID       string
+	Channel      model.AuthenticatorOOBChannel
+	LoginID      string
```

Comment:
It should not take LoginID. The target to resend to should be stored in the token already.